libs/application/templates/estate/src/lib/dataSchema.ts

```diff
@@ -132,6 +133,14 @@ export const estateSchema = z.object({
           path: ['email'],
         },
       )
+      .refine(
+        ({ enabled, relationWithApplicant }) => {
+          return enabled ? relationWithApplicant !== '' : true
+        },
+        {
+          path: ['relationWithApplicant'],
+        },
+      )
```

Comment:
Will this work for all four paths of the application?
If the `relationWithApplicant` field isn't visible, the mapper will still default relationWithApplicant to `''`, right?
Is it possible this will make validation fail while this field is invisible?

Reply:
oh, i forgot to remove it, thanks MAN